src/schema/v2/artwork/context.ts

```diff
@@ -58,10 +58,9 @@ const Context: GraphQLFieldConfig<any, ResolverContext> = {
         return assign({ context_type: "Fair" }, fair)
       })
 
-    const show_promise = relatedShowsLoader({
-      artwork: [id],
+    const show_promise = showsLoader({
+      artwork: id,
       size: 1,
-      active: false,
```

Comment:
This was a default so never needed to be specified, plus the shows endpoint doesn't accept this as a query param anyway. Otherwise all the query params lined up 1-1.